### DISCUSSION TITLE
@Transactional 비교! 러닝하이는 어떤 걸 사용할까요

### DISCUSSION THREAD

**Discussion by silverpoodle** [cca-ffodregamdi/running-hi-back-v2#32] [Show and Tell]
## 1. org.springframework.transaction.annotation의   @ Transactional

`javax`는 자바 표준 확장 패키지를 뜻합니다. = 자바에서 기본 제공되는 것

### 구현 기능

- Transaction.type
- Rollback

<br/>

## 2. javax.transaction의   @ Transactional

`org.springframework`에서 제공 = **Container가 관리**해준다는 것

### 구현기능

- propagation
- isolation
- timeout
- readOnly
- rollback
등등...

_Java에서 제공하는 Transaction 기능에서 Spring에서 Framework를 위한 기능을 보충_

<br/>

개인적으로는 더 많은 옵션과 더 많은 버전을 지원하는 @org.springframework.transaction.annotation.Transactional를 사용하는 것이 더 좋을 것 같습니다.

**Comment by numerical43** [comment]
넵 저도 @org.springframework.transaction.annotation.Transactional 쪽이 더 좋을 것 같습니다!

**Reply by numerical43** [reply]
그리고 위아래 제목이 바뀐 것 같아요ㅎㅎ

**Comment by Subak-Uncle** [comment]
오 정리 감사합니다:) 더 다양한 기능을 지원해주는 SpringFramework가 구미가 당기는군요!
